testing/bdd/features/deduplicator/deduplicator-blackbox.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Deduplicator removes duplicated style and region elements
```

Comment:
We don't need to have "-blackbox" in the filename - it's a BDD so that's what it does.